### ISSUE TITLE
Dedicated master-eligible nodes appear as client nodes

### ISSUE THREAD

**Issue by zmalik** [mobz/elasticsearch-head#260]
Hi, 

We have a elasticsearch cluster. With dedicated master-eligible nodes, which when aren't in master role. Are being shown up as client node. Which isn't true.

So basically the logic checks when a node isn't in master role, even if it is a master-eligible node. It will appear as Client node, as far as if it isn't a data node.

So for dedicated master-eligible nodes with following configuration in their elasticsearch.yml:
```
node.master: true 
node.data: false
```
When they aren't in master role, they will be appear as client node.

As a solution I think we can save inside node object the value of `cluster.attributes.master`. And depending on that value decide if its a client node, or just another master-eligible node without data.

If the solution seems fine I can even prepare a simple PR.

Thanks

**Comment by mobz** [comment]
Yes, I see the issue, We identify what elastic.co describe as coordinating nodes (which are non-data non-master), but differentiate between coordination nodes that are master eligible or not.

Hmm.

There is also ingest specific nodes. So you can define a cluster where some node ingest and some don't.

How would you propose to indicate that a node is master-eligible but not master? 
I don't think it's appropriate to have master and master-eligible have the same icon.

It should be possible to determine master-eligible based on the setting node.master = true node.master_node = false ?

I use icons from http://fontawesome.io/icons/

**Comment by mobz** [comment]
I wonder if the best solution is detecting how `configured` a cluster is and adjusting the number of icons based on that?

As an example, if all nodes are master = true, data = true, ingest = true, it's a basic cluster and you just show the difference between master and data nodes

If some nodes are configured for master = false, then you show an extra icon for master-eligible
If some nodes are ingest and some note, then you show an extra icon for ingest nodes?

So the complexity of the icons increases for more finely tuned clusters?
